app/views/admin/questions/_question.html.erb

```diff
@@ -0,0 +1,15 @@
+<tr class="h-50 body-row">
+  <td><%= question.title %></td>
+  <td><%= question.description %></td>
+  <td><%= question.score %></td>
+  <td>
+    <%= link_to '&#x270D;'.html_safe, edit_admin_question_path(question), class: 'btn btn-info', title: 'edit' %>
```

Comment:
Start using icons

Reply:
https://trello.com/c/lUzuPSSH/17-icons-instead-of-unicode